Add Redfish to Power Management (#629)
uyuni-project/uyuni-docs: modules/client-configuration/pages/power-management.adoc

You can power on, power off, and reboot clients using the {productname} {webui}.

This feature uses either the IPMI or Redfish protocol and is managed using a Cobbler profile.

> The selected client must have the protocol enabled, and be configured to use a power management server.

Reviewer: ```suggestion
The selected client must have a power management controller supporting one of these protocols.
```

> For Redfish, ensure you have a valid SSL connection between the client and the {productname} Server.

Reviewer: ```suggestion
For Redfish, ensure you can establish a valid SSL connection between the client and the {productname} Server.
```
